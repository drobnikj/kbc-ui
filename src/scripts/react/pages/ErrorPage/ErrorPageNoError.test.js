import React from 'react';

import ErrorPage from '../ErrorPage';

jest.mock('../../../stores/RoutesStore', () => {
  return {
    getError: () => {
      return {
        getText: () => 'error-text',
        getExceptionId: () => 'error-exception-id'
      };
    }
  };
});

describe('<ErrorPage />', function() {
  it('should render with no props', function() {
    shallowSnapshot(
      <ErrorPage />
    );
  });
});
