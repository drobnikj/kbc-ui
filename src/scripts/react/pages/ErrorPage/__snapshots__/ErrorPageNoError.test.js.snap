exports[`<ErrorPage /> should render with no props 1`] = `
<div
  className="container-fluid">
  <div
    className="kbc-main-content">
    <Alert
      bsClass="alert"
      bsStyle="danger"
      closeLabel="Close alert">
      <div>
        <p>
          error-text
        </p>
        <p>
          Exception id: error-exception-id
        </p>
      </div>
    </Alert>
  </div>
</div>
`;
